### PR TITLE
Fix for Inventory deletion

### LINKED ISSUE #1003: Items not saving in inventories of either base, player or storage
I have this issues that when ever I close the server and start the server all the item in the inventories even storage all disapear, for some reason only the repair tool are remain and other contrustion like mobile station contrustion are still remain

### PR DESCRIPTION
**Pull request by killzoms**
fixes #1003 
closes #1066

### CODE REVIEW

File: NitroxPatcher/Patches/Dynamic/uGUI_OnApplicationQuit_Patch.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+        public static void Prefix()
+        {
+            IMultiplayerSession multiplayerSession = NitroxServiceLocator.LocateService<IMultiplayerSession>();
+            multiplayerSession.Disconnect();
+        }
+
+        public override void Patch(HarmonyInstance harmony)
+        {
+            PatchPrefix(harmony, TYPE.GetMethod(TARGET_METHOD, BindingFlags.NonPublic | BindingFlags.Instance));
```
Comment (Jannify) [inline NitroxPatcher/Patches/Dynamic/uGUI_OnApplicationQuit_Patch.cs:22]: Maybe change so it's consistent with other patches code style wise.

File: Packages/manifest.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+  "dependencies": {
```
Comment (Jannify) [inline Packages/manifest.json:2]: What's this file doing here?

Comment (killzoms) [reply Packages/manifest.json:2]: it appeared out of the blue so I left it there...

**Comment by killzoms** [comment]
@Coding-Hen due to the overall "fix" being tiny felt it would be better to include the Autofac update with the patch instead of a new PR, also updating Autofac has some added benefits as well, besides matching the .Net framework version

File: Packages/manifest.json (hunk 2 above)
Comment (Jannify) [reply Packages/manifest.json:2]: That file is usually generated inside unity projects but here it's in the root folder and not `/NitroxUnity`. So it's save to delete.

**Pull request merged**